.github/workflows/files-changed.yml

```diff
@@ -64,9 +64,12 @@ jobs:
               - "**/*.md"
               - "docs/**"
               - ".markdownlint.yaml"
+              - "package.json"
+              - "package-lock.json"
```

Comment:
I think those two are a bit weird for the `docs` category.
If I see that correctly, it'll mean that any JS dependency update or new JS feature will be marked as `kind/docs`.

Reply:
Feel free to add or remove the `kind/docs`

Reply:
This is not the labels config, this is the actions config. `mardownlint-cli` version is defined in these files, if those change we need to run the docs workflow to confirm it still works.